qr-code/src/main/java/com/adyen/checkout/qrcode/internal/ui/view/FullQRCodeView.kt

```diff
@@ -152,12 +153,8 @@ internal class FullQRCodeView @JvmOverloads constructor(
     }
 
     @StringRes
-    private fun getMessageTextResource(paymentMethodType: String?): Int? {
-        return when (paymentMethodType) {
-            PaymentMethodTypes.PAY_NOW -> R.string.checkout_qr_code_pay_now
-            PaymentMethodTypes.UPI_QR -> R.string.checkout_qr_code_upi
-            else -> null
-        }
+    private fun getMessageTextResource(paymentMethodType: String): Int? {
```

Comment:
I think this should be done in delegate and we should have a field in `QRCodeOutputData` for the resourceId. Here we should just set it to keep view as simple as possible.